### PR TITLE
Add a way to build a debian with the spotipy package

### PR DESCRIPTION
**Pull request by asetapen**
- Bumps version number
- Adds a script for creating a debian

### Instructions for creating debian
- Add the `upboard-dev` packages from the [Hacking on a Robot wiki](https://mayfieldrobotics.atlassian.net/wiki/spaces/SOF/pages/32211351/Hack+on+a+robot)
- Add ruby packages for `fpm`:
```bash
sudo apt-get install ruby ruby-dev
```

- Install `fpm`:
```bash
gem install --no-ri --no-rdoc fpm
```

- Run the script that creates the debian:
```bash
./helpers/build_debian.sh
```

### CODE REVIEW

**Comment by asetapen** [comment]
Getting an error on `./build_debian.sh`: 
```
Process failed: easy_install failed (exit code 1). Full command was:["easy_install", "-i", "https://pypi.python.org/simple", "--editable", "-U", "--build-directory", "/tmp/package-python-build-47525ae25e3f3f871c8e685eddbf1add889de08bebbd1364c2770a8a0a51/setup.py", "setup.py"] {:level=>:error}
```

File: helpers/build_debian.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+fpm \
+  --input-type python \
+  --output-type deb \
+  --architecture native \
+  --python-install-bin "/usr/bin" \
+  --python-install-lib "/usr/lib/python2.7/dist-packages" \
+  --depends "requests, six python-mayfield-tools" \
```
Comment (po1) [inline helpers/build_debian.sh]: `python-requests, python-six, python-mayfield-tools`

**Review by po1 — APPROVED** [review]
Nice! Can we add that to the `.travis.yml` to make sure that we never break the packaging?

**Comment by po1** [comment]
Ah, we don't have travis integration for this.. never mind then :stuck_out_tongue:

File: helpers/build_debian.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+fpm \
+  --input-type python \
+  --output-type deb \
+  --architecture native \
+  --python-install-bin "/usr/bin" \
+  --python-install-lib "/usr/lib/python2.7/dist-packages" \
+  --depends "python-requests, python-six, python-mayfield-tools" \
```
Comment (po1) [inline helpers/build_debian.sh]: why do we need mayfield-tools?

**Review by spmaniato — APPROVED** [review]
LGTM :+1: @asetapen @po1 Looking at this PR, installing `python-six` and `python-requests` is no longer necessary, right? We have them from `pip`

**Review by spmaniato — CHANGES_REQUESTED** [review]
Hmm, `fpm` is picking up the `install_requires` from `setup.py` I think we should remove that section and rely on `pip` packages for the dependencies, not Debian packages.

PS. `fpm` has a `--python-disable-dependency` option we can use if you don't wanna modify `setup.py`

**Review by po1 — APPROVED** [review]

**Review by spmaniato — APPROVED** [review]
Not that it matters anymore, but Debian looks good:

```
$ dpkg-deb --info python-spotipy_2.4.5_amd64.deb 
 new debian package, version 2.0.
 size 25342 bytes: control archive=869 bytes.
     322 bytes,    11 lines      control              
    1296 bytes,    14 lines      md5sums              
 Package: python-spotipy
 Version: 2.4.5
 License: MIT_License
 Vendor: Mayfield Robotics
 Architecture: amd64
 Maintainer: Adam Setapen <adam@mayfieldrobotics.com>
 Installed-Size: 93
 Section: default
 Priority: extra
 Homepage: https://github.com/MayfieldRoboticsPublic/spotipy
 Description: simple client for the Spotify Web API
```

**Pull request merged**